ansible/roles/jenkins/tasks/main.yml

```
@@ -0,0 +1,46 @@
+---
+# tasks file for jenkins
+
+- name: Install tools
+  ansible.builtin.import_tasks:
+    file: tools.yml
+
+- name: Install Jenkins
+  ansible.builtin.apt:
+    name: "{{ java_version }}"
+    state: present
+
+- name: Add jenkins apt key
+  ansible.builtin.apt_key:
+    url: "{{ jenkins.jenkins_key }}"
+    state: present
+
+- name: Add jenkins apt repository
+  ansible.builtin.apt_repository:
+    repo: "{{ jenkins.jenkins_repo }}"
+    state: present
+    become: true
+
+- name: Update apt cache
+  ansible.builtin.apt:
+    update_cache: yes
```

yaml[truthy]: Truthy value should be one of [false, true]
```suggestion
    update_cache: true
```